.github/workflows/integration.yml

```diff
@@ -83,6 +83,62 @@ jobs:
           target/
         key: ${{ steps.restore-cargo-cache.outputs.cache-primary-key }}
 
+  arm64-test:
+    name: Tests (arm64)
+    runs-on: [self-hosted, linux, arm64]
```

Comment:
If we also have `x86_64` self-hosted runners, can we just add ARM and x86 to the strategy matrix and reuse the workflow?

Reply:
I'm not entirely sure, but from examples in [github docs](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-using-a-multi-dimension-matrix) I believe we can.